src/App.vue

```diff
@@ -17,6 +18,7 @@ import TheErrorModal from "./components/TheErrorModal";
 import TheMap from "./components/TheMap";
 import TheProgressCircle from "./components/TheProgressCircle";
 import TheWarningAlert from "./components/TheWarningAlert";
+import Timelapse from "./components/Timelapse"
```

Comment:
We should change the name of this component to be at least two words (`TheTimelapse` perhaps?); see this essential priority style point: https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential

Reply:
Sure, TheTimelapse sounds good